servers/dsub/jobs/test/test_jobs_controller_local.py

```diff
@@ -33,6 +34,9 @@ def setUp(self):
         # Create logging directory
         self.log_path = '{}/logging'.format(self.testing_root)
         os.mkdir(self.log_path)
+        self.test_token_label = {
```

Comment:
This is duplicated, please call `super()` here instead and put this logic into the base_test_cases class